### ISSUE TITLE
Unable to click checkboxes or buttons/links if row height is larger than viewport

### ISSUE THREAD

**Issue by jonw-bb** [handsontable/handsontable#5984]
### Description
If a row height is larger than the viewport height, selecting a cell on that row will cause it to scroll to the bottom.

If the cell contains a call to action such as a checkbox, button or anchor link, it will not get fired.

This has been raised elsewhere before - https://forum.handsontable.com/t/gh-4173-unable-to-select-checkbox-when-text-is-large-multi-line-input-in-column/1890

But I believe it was closed and linked to another issue which I do not believe to be the same thing.

I am aware that I could call afterSelection hook and set preventScrolling to true. But the problem here is that navigating the table with a keyboard where the view requires a scroll to the selected cell no longer works.

### Steps to reproduce

1. Make sure the height of a row is more than viewport height and requires scrolling (the second row in the JSFiddle below).
2. Click on the checkbox on the same row in the last column.

### Demo

https://jsfiddle.net/4zxbdv0s/

### Your environment
* Handsontable version: 6.2.2
* Browser Name and version: Chrome 73.0.3683.103
* Operating System: MacOS High Sierra

**Comment by AMBudnik** [comment]
You're right. The closed issue and the one reported by you are two different cases.
I have replicated the behavior and I can understand why this can be confusing. We do not expect to be scrolled down when we click the checkbox.

However, it is a distinguished example so the fix might be a something more like a workaround.
I'll investigate this subject further and share my thoughts.

**Comment by AMBudnik** [comment]
I have simplified the example https://jsfiddle.net/AMBudnik/h45mgrkv/ - we just get the default scroll once a cell is selected.

**Comment by AMBudnik** [comment]
Happens the same in v `7.4.2` https://jsfiddle.net/AMBudnik/mv5kqrLd/
We need to scroll up and then we are able to select that checkbox.

**Comment by AMBudnik** [comment]
This issue remains the same in v8.2.0 https://jsfiddle.net/handsoncode/awhzs3k5/
When we click the checkbox we are selecting the cell and the viewport is scrolled to show the bottom of the selection. That's why we need to scroll up again to select the checkbox.

**Comment by AMBudnik** [comment]
The following issue was altered. We are no longer scrolled (that much). Updated demo https://jsfiddle.net/wLbz1v5m/ (Handsontable 12.1.3).

But now we cannot select the checkbox with a single click. This issue can be related to https://github.com/handsontable/handsontable/issues/9925 where we also need to double-click the checkbox to change its state but we set up `fixedColumnsLeft`.

**Comment by budnix** [comment]
The issue is fixed in v17 https://jsfiddle.net/budnix/dm8spj4w/
